### PR TITLE
Updates Religion Lists + Adds Religious Paraphernalia

### PR DESCRIPTION
**Pull request by Cerebulon**
- Changes the religion selection in character setup to be in line with religions in lore and the wiki.
- Changes holy book names based on Chaplain religion input to those too.
- Adds black and white candles in addition to existing red. (Small candles were also already sprited and are now in the code and can be adminspawned.)
- Adds religion icons/statues with 29 selectable sprite variations (Similar to the detective's gun)
- Icons and new candles are in the chaplain closet.

Icon Sprites:
https://i.gyazo.com/2155a60b053deda9514372b61b86a4ad.png

### CODE REVIEW

**Comment by elgeonmb** [comment]
thanks cere

File: code/game/objects/items/weapons/candle.dm [hunk 1]
```diff
@@ -68,6 +69,27 @@
...
+/obj/item/weapon/flame/candle/white
+	name = "white candle"
+	desc = "a white pillar candle. Its specially-formulated fuel-oxidizer wax mixture allows continued combustion in airless environments."
+	icon = 'icons/obj/candle.dmi'
+	icon_state = "whitecandle"
+	w_class = ITEMSIZE_SMALL
+
+/obj/item/weapon/flame/candle/black
+	name = "black candle"
+	desc = "a white pillar candle. Ominous."
```
Comment (elgeonmb) [inline code/game/objects/items/weapons/candle.dm]: should be black

**Comment by SubberTheFabulous** [comment]
>removed satanism
Me is unhappiness.

**Comment by Cerebulon** [comment]
You can still enter it manually it's just not a major religion. Also one of the statue options is literally baphomet and you can light black candles so go ham.

**Pull request merged**